search/tests/api/stac.spec.js

```diff
@@ -76,3 +80,157 @@ describe('STAC Search', () => {
     });
   });
 });
+
+describe('STAC Search Params', () => {
+  let request, response;
+
+  beforeEach(() => {
+    response = createMockResponse(200, { feed: { entry: [] } });
+    axios.get = jest.fn(async () => {
+      return Promise.resolve({
+        headers: { 'cmr-hits': 0 },
+        data: { feed: { entry: [] } },
+        json: (v) => JSON.stringify(v, null, 2)
+      });
+    });
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  describe('datetime', () => {
+    it('should query with range when given a single datetime', async () => {
+      request = createRequest({
+        params: { providerId: 'LPDAAC' },
+        query: { datetime: '2020-11-02T00:00:00Z' }
```

Comment:
If there aren't already tests for handling malformed datetimes, it's worth adding them, as well as ones for something like:
`query: { datetime: '2020-11-02T00:00:00Z,' }` or `query: { datetime: '/2020-11-02T00:00:00Z' }`